fsproxy/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'nva.identity.service.java-application-conventions'
+    id 'nva.identity.service.dynamodblocal'
+
+}
+
+dependencies {
+
+    implementation libs.nva.core
+    implementation libs.nva.json
+    implementation libs.bundles.jackson
+    implementation 'joda-time:joda-time:2.10.14'
```

Comment:
Do not import a library to only get the current year. Plain java can do that `LocalDate.now().getYear()`